### DISCUSSION TITLE
Removing LEQ

### DISCUSSION THREAD

**Discussion by htxwn** [Falcosc/enable-loudness-equalisation#25] [Q&A]
Hello, How I do remove the settings done by the LEQ script? Apologies if I didn't see instructions, but I'm not locating where to basically "uninstall" the setting. I can't use my headphones with any EQ at all anymore and its just far too loud and obnoxious.

**Comment by Falcosc** [comment]
Untick in sound settings, like in the screenshot. The script is only unhiding and selecting the option. So you can just unselect it.

**Comment by DanieleRiccelli** [comment]
Hi i install on my front panel audio jack (realtek) but if go to the option i don't have the flag to unselect the EQ, some tips?

**Comment by Falcosc** [comment]
Backup and remove the Registry folder of your audio device, which got adjusted. Drivers will restore the default settings if the folder disappeared during reboot.

But Backup in case you have a stupid driver installed which isn't capable.

If you have trouble finding the folder, the path of what got added can be found in %TMP%\SoundEnhancementsTMP.reg


Or just remove the key "{fc52a749-4be9-4510-896e-966ba6525980},3" which contains the enable/disable state
Or remove all keys which got adjusted
```
"{d04e05a6-594b-4fb6-a80d-01af5eed7d1d},1"="{62dc1a93-ae24-464c-a43e-452f824c4250}" ;PreMixEffectClsid activates effects
"{d04e05a6-594b-4fb6-a80d-01af5eed7d1d},2"="{637c490d-eee3-4c0a-973f-371958802da2}" ;PostMixEffectClsid activates effects
"{d04e05a6-594b-4fb6-a80d-01af5eed7d1d},3"="{5860E1C5-F95C-4a7a-8EC8-8AEF24F379A1}" ;UserInterfaceClsid shows it in ui
"{d04e05a6-594b-4fb6-a80d-01af5eed7d1d},5"="{62dc1a93-ae24-464c-a43e-452f824c4250}" ;StreamEffectClsid
"{d04e05a6-594b-4fb6-a80d-01af5eed7d1d},6"="{637c490d-eee3-4c0a-973f-371958802da2}" ;ModeEffectClsid
"{fc52a749-4be9-4510-896e-966ba6525980},3"=hex:0b,00,00,00,01,00,00,00,ff,ff,00,00 ;enables loudness equalisation
"{9c00eeed-edce-4cd8-ae08-cb05e8ef57a0},3"=hex:03,00,00,00,01,00,00,00,$releaseTimeStr,00,00,00 ;equalisation release time 2 to 7
```

**Reply by Wartogh** [reply]
Sorry to bother you with this, but yesterday I installed this LEQ and it messed my headphone. I think it conflicted with the Razer App Engine or something because the 7.1 surround sound disappeared and now there's a new headphone option on the voice and video options on Discord. I Think I removed the keys you mentioned like the one which contains the enable/disable state and the other 7 you mentioned after and restarted my computer. Although the "enhancements" tab disappeared from the playback device properties, I still can't use my "7.1 Surround Sound" device which was my headset, it has no sound, and now it seems Windows sees my headset as two separated devices (a phone and a microphone)... I'm missing something or by removing all these keys, everything was supposed to go back to what it was before I installed the LEQ? Maybe I have to remove the Registry folder of my audio device for the Drivers restore the default settings, but I don't know how to find it... Can you help me, please?

**Reply by Falcosc** [reply]
Yes, removing the folder does restore it.

**Reply by Wartogh** [reply]
> Yes, removing the folder does restore it.

And how can I find the Registry folder of my audio device?

**Reply by Falcosc** [reply]
```
reg query HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\MMDevices\Audio\Render /s /f $playbackDeviceName /d
```

**Reply by Wartogh** [reply]
I pasted this command on the PowerShell and ir returned the following:

```
PS C:\Windows\system32> reg query HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\MMDevices\Audio\Render /s /f $playbackDeviceName /d
 
End of search: 0 match(es) found.
```

**Reply by Wartogh** [reply]
Hi! I tried to restore my system and now everything is ok, it went back for before I installed LEQ, thanks for your support!

**Comment by DanieleRiccelli** [comment]
Thanks you
